administrator-guides/federation/README.md

```diff
@@ -1,11 +1,88 @@
 # Federation
 
+## Introduction
+
+Federation allows communication with other Rocket.Chat servers.
+
+When you open Federation's config screen, you will see this:
+
+![Initial Federation Screen](./initial.png){:style="width='100%' height='auto'"}
```

Comment:
seems `{:style ...}` isn't a valid GitHub Markdown, as seen [here](https://github.com/RocketChat/docs/blob/f7748c67de2cf80336c275f0e96df8174459fe58/user-guides/federation/README.md)

but on GitHub (and hopefully our website) that's the default style for images =) so we can remove all styles.

Reply:
Cool!